test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -85,13 +85,13 @@ def test_ec2_pytorch_inference_neuronx(pytorch_inference_neuronx, ec2_connection
 @pytest.mark.model("densenet")
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_ec2_pytorch_inference_gpu(
-    pytorch_inference, ec2_connection, region, gpu_only, ec2_instance_type
+    stability_pytorch_inference, ec2_connection, region, gpu_only, ec2_instance_type
```

Comment:
this is a test, will revert prior to merge

Reply:
Thanks! Expected behavior would be that this would not run on PT and would run on SAI-PT, right?